### ISSUE TITLE
Validator - Output file named "null_pointset.out", "null_system.out"

### ISSUE THREAD

**Issue by noursaidi** [faucetsdn/udmi#404]
Validator output files are named "null_pointset.out", "null_system.out", but the messages themselves are correctly named:

```
nournew@macbook-2 udmi % ls -1 sites/udmi_site_model/out/devices/AHU-1 
event_pointset.attr
event_pointset.json
event_system.attr
event_system.json
null_pointset.out
state.attr
state.json
state_pointset.attr
state_pointset.json
state_pointset.out
state_system.attr
state_system.json
state_system.out
state_update.out
```

To recreate, Pubber as the device with the following run command - `bin/pubber sites/udmi_site_model PROJECT_ID AHU-1 123 noHardware extraPoint=extra_point extraField=extra_field `

**Comment by grafnu** [comment]
I haven't been able to reproduce the null_ part of this, so either 1) it's been fixed already, or 2) I'll need some more guidance on how to reproduce.

**Comment by noursaidi** [comment]
@grafnu this example fixes this issue for me - https://github.com/grafnu/udmi/pull/3/files 

The reproduction steps validate any message and the output results (`.out` file) for `event` messages is `null_pointset`, `null_system` etc.

**Comment by grafnu** [comment]
Still unable to reproduce... I'd like to at least understand the problem before tweaking things.  See below for my test run -- I have a generic pubber (no options) running in another window. This is on the current master branch:

```
peringknife@peringknife-glaptop4:~/udmi$ bin/validator sites/udmi_site_model bos-peringknife-dev
Building validator...
Running tools version 1.3.14-41-gf9dce437
No subscription supplied, assuming UDMI reflector.
Application default credentials auth type authorized_user
Cleaning output directory /home/peringknife/udmi/sites/udmi_site_model/out/
Executing validator schema reflect...
java -jar validator/build/libs/validator-1.0-SNAPSHOT-all.jar -p bos-peringknife-dev -s /home/peringknife/udmi/sites/udmi_site_model
Loaded 4 expected devices
Generating report file in /home/peringknife/udmi/sites/udmi_site_model/out/validation_report.json
Loading reflector key file from /home/peringknife/udmi/sites/udmi_site_model/validator/rsa_private.pkcs8
[main] INFO com.google.bos.iot.core.proxy.MqttPublisher - ZZ-TRI-FECTA token expiration sec 3600
[main] INFO com.google.bos.iot.core.proxy.MqttPublisher - ZZ-TRI-FECTA creating client projects/bos-peringknife-dev/locations/us-central1/registries/UDMS-REFLECT/devices/ZZ-TRI-FECTA on ssl://mqtt.googleapis.com:8883
[main] INFO com.google.bos.iot.core.proxy.MqttPublisher - ZZ-TRI-FECTA creating new jwt
[main] INFO com.google.bos.iot.core.proxy.MqttPublisher - ZZ-TRI-FECTA connecting to mqtt server
[main] INFO com.google.bos.iot.core.proxy.MqttPublisher - ZZ-TRI-FECTA adding subscriptions
[main] INFO com.google.bos.iot.core.proxy.MqttPublisher - ZZ-TRI-FECTA done with setup connection
Entering message loop on bos-peringknife-dev/us-central1/UDMS-REFLECT/ZZ-TRI-FECTA
Processing device #1/4: AHU-1/event_pointset
Warning: Nashorn engine is planned to be removed from a future JDK release
Validation complete AHU-1/event_pointset
Processing device #1/4: AHU-1/event_system
Error validating schema: While converting to json node: 1 schema violations found
^CValidation complete, exit 130
peringknife@peringknife-glaptop4:~/udmi$ find sites/udmi_site_model/out -name \*null\*
peringknife@peringknife-glaptop4:~/udmi$
```

**Comment by noursaidi** [comment]
Try with a pub/sub subscription. The root cause is the value of `subType` is `null` for telemetry messages.

Just tried with reflector and it works, presumably because it sets these attributes.

**Comment by grafnu** [comment]
Ok -- yes, can reproduce with that. The pubsub and reflector paths are
slightly different, so not too surprising!

**Comment by noursaidi** [comment]
Fixed by #439
